app/src/main/java/com/github/polypoly/app/base/menu/lobby/GameMode.kt

```diff
@@ -4,9 +4,14 @@ package com.github.polypoly.app.base.menu.lobby
  * Represent the different game modes
  */
 enum class GameMode(val description: String, private val title: String) {
-    LAST_STANDING("In this game mode, the game lasts until only one player is left standing 😎", "Last Standing"),
+    LAST_STANDING("In this game mode, the game lasts until only one player is left standing 😎"
+        , "Last Standing"),
     RICHEST_PLAYER("In this game mode, the game lasts for a fixed number of turn and the winner"
-        + "is the player with the greatest net worth in the end 🤑", "Richest Player");
+        + "is the player with the greatest net worth in the end 🤑"
+        , "Richest Player"),
+    LANDLORD("In this game mode, all the buildings are equally distributed between the players at the beginning of the game. " +
```

Comment:
nitpick: you can precise that they are randomly distributed

Reply:
nitpick of nitpick: "préciser" = to specify or to clarify in this context, precise is not a verb ;)